Doc/library/ast.rst

```diff
@@ -146,6 +146,95 @@ Node classes
     Snakes <https://greentreesnakes.readthedocs.io/en/latest/>`__ project and
     all its contributors.
 
+
+.. _ast-root-nodes:
+
+Root nodes
+^^^^^^^^^^
+
+.. class:: Module(body, type_ignores)
+
+   A Python module.
+
+   *body* is a :class:`list` of the module's :ref:`ast-statements`.
+   *type_ignores* is a :class:`list` of the module's type ignore comments,
+   see :func:`ast.parse` for more details.
+
+   .. doctest::
+
+        >>> print(ast.dump(ast.parse('x = 1'), indent=4))
+        Module(
+            body=[
+                Assign(
+                    targets=[
+                        Name(id='x', ctx=Store())],
+                    value=Constant(value=1))],
+            type_ignores=[])
+
+
+.. class:: Expression(body)
+
+   A single Python expression.
+
+   *body* is a single :ref:`ast-expressions` node.
```

Comment:
```suggestion
   *body* is a single node,
   one of the :ref:`expression types <ast-expressions>`.
```

* Make the sentence grammatical with the injected reference
* Clarify that we're referring to a set of types rather than a literal `Expression` type (i.e. this node class)